README.md

```diff
@@ -12,6 +12,14 @@ guides, and instructions on building from source.
 [![CI Status](https://github.com/openxla/iree/actions/workflows/ci.yml/badge.svg?query=branch%3Amain+event%3Apush)](https://github.com/openxla/iree/actions/workflows/ci.yml?query=branch%3Amain+event%3Apush)
 [![IREE Discord Status](https://discordapp.com/api/guilds/689900678990135345/widget.png?style=shield)](https://discord.gg/26P4xW4)
 
+#### Set Up IREE
+
+```bash
+git clone https://github.com/openxla/iree
+cmake -G Ninja -B ../iree-build/ .
+cmake --build ../iree-build/
+```
```

Comment:
I'm hesitant to include the most basic build instructions here without the associated warnings about it being slow and unnecessary for some users (this builds the compiler in debug and can take 30+ minutes on some machines).

How about linking to https://iree.dev/building-from-source/getting-started/ and https://iree.dev/reference/bindings/python/ instead? (I can draft some phrasing)

Reply:
How about this:

```markdown
## Installation and Setup

* Instructions for installing and using IREE's Python packages are available at
https://iree.dev/reference/bindings/python/.
* To build from source, refer to
https://iree.dev/building-from-source/getting-started/.
```

(maybe after the `#### Project Status` section, or after `#### Related Project Channels`)

Reply:
There are some packages needed as well... at least for linux. For old ubuntu, I think you even need to build cmake yourself. Linking it to a proper doc makes sense to me.

Reply:
@ScottTodd that looks lovely.

I didn't have to build cmake myself (used the apt install one) but I did have to install ncurses, which wasn't mentioned in any of the docs i think.

Reply:
I would love more of a faq style page that we could easily add to instead of the current docs which have no place for practical, case by case knowledge...

All of these cases can and do happen.

Reply:
> I would love more of a faq style page that we could easily add to instead of the current docs which have no place for practical, case by case knowledge...
> 
> All of these cases can and do happen.

Loosely organized info could go next to https://iree.dev/developers/building/cmake-with-ccache/ on the developers section, or just at the bottom of the main building from source page.

I just want to be careful about putting too many system-specific instructions on the main path that all developers are expected to read.